packages/dropdown/src/css/index.ts

```diff
@@ -169,7 +169,9 @@ export default {
   '.psds-dropdown__menu': {
     position: 'fixed',
     zIndex: 980,
-    marginTop: layout.spacingXXSmall
+    marginTop: layout.spacingXXSmall,
+    maxHeight: 400,
+    overflowY: 'scroll'
```

Comment:
```suggestion
    overflowY: 'auto'
```

I'd suggest this change.  Works for scrolling when the max height is exceeded.  Looks nicer for the (hopefully) common case of no scroll.

What do you think?